ReleaseNotes.md

```diff
@@ -8,6 +8,53 @@ information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeo
 additional questions or comments.
 
 ## Release History
+### 01/31/2024 -
+##### Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:3.1.17 (linux)
+##### Version mcr.microsoft.com/azuremonitor/containerinsights/ciprod:win-3.1.17 (windows)
+##### Release branch: https://github.com/microsoft/Docker-Provider/commits/release-ciprod-3.1.17/
+##### Current dependencies
+- Linux
+  - [CBL-Mariner 2.0.20231004](https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20231004-2.0)
```

Comment:
CBL Mariner version is  2.0.20240123  https://github.com/microsoft/CBL-Mariner/releases/tag/2.0.20240123-2.0
![image](https://github.com/microsoft/Docker-Provider/assets/20712167/6372041f-4056-40d6-b0b5-ca0cde4bf8db)

Reply:
added